test/objcopy.c

```diff
@@ -1678,11 +1678,17 @@ compare_groups(hid_t gid, hid_t gid2, hid_t pid, int depth, unsigned copy_flags)
                     case H5O_TYPE_MAP:
                         HDassert(0 && "maps not supported in native VOL connector");
 
+                        /* clang complains about implicit fallthrough here and
+                         * our usual attributes and fall-through comments don't
+                         * quiet the compiler.
+                         */
+                        H5_CLANG_DIAG_OFF("implicit-fallthrough")
```

Comment:
Apparently clang doesn't respect adding the fall-through attribute to empty blocks